client/my-sites/posts/post.jsx

```diff
@@ -375,13 +280,7 @@ module.exports = React.createClass( {
 					{ this.getHeader() }
 					{ this.getPostImage() }
 					{ this.getContent() }
-					<footer className="post__info">
-						<PostRelativeTimeStatus post={ this.props.post } link={ this.getContentLinkURL() } target={ this.getContentLinkTarget() } onClick={ this.analyticsEvents.dateClick }/>
-						{
-							// Only show meta items for non-drafts
-							( this.props.post.status === 'draft' ) ? null : this.getMeta()
-						}
-					</footer>
+					<PostActions { ...{ site, post: this.props.post, toggleComments: this.toggleComments } } />
```

Comment:
instead of passing `site` down here could I encourage us to pass a simple-data reference to a site and allow `connect#mapStateToProps()` to provide the actual site data to `PostActions`?

note also how in `PostActions` we're coupling the data of the `site` object in Calypso with the structure of the data the API returns (as indicated by `comment_count` instead of `commentCount` - we would ideally get the comment count from state directly)